docs/_includes/generated/kubernetes_config_configuration.html

```diff
@@ -18,7 +18,7 @@
             <td><h5>kubernetes.cluster-id</h5></td>
             <td style="word-wrap: break-word;">(none)</td>
             <td>String</td>
-            <td>The cluster-id, which should be no more than 45 characters, is used for identifying a unique Flink cluster. If not set, the client will automatically generate it with a random ID.</td>
+            <td>The cluster-id, which should be no more than 45 characters, is used for identifying a unique Flink cluster. The id must only contain lowercase alphanumeric characters, "-" or ".". If not set, the client will automatically generate it with a random ID.</td>
```

Comment:
I am sorry for providing an inaccurate information. Actually, `.` could not be supported here. And if the `kubernetes.cluster-id` is set to `k8s-ha-app-1.test`. We will get the following exception.

```
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://192.168.64.19:8443/api/v1/namespaces/default/services. Message: Service "k8s-ha-app-1.test-rest" is invalid: metadata.name: Invalid value: "k8s-ha-app-1.test-rest": a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?'). Received status: Status(apiVersion=v1, code=422, details=StatusDetails(causes=[StatusCause(field=metadata.name, message=Invalid value: "k8s-ha-app-1.test-rest": a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?'), reason=FieldValueInvalid, additionalProperties={})], group=null, kind=Service, name=k8s-ha-app-1.test-rest, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=Service "k8s-ha-app-1.test-rest" is invalid: metadata.name: Invalid value: "k8s-ha-app-1.test-rest": a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?'), metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Invalid, status=Failure, additionalProperties={}).
```

Reply:
Alright, I will update it.